roles/ovirt-ova-pack/files/pack_ova.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+
+import os
```

Comment:
At the end of the day, this file will be stored in ovirt-engine, where we run pep8 checks, can you please re-sort alphabetically the imports?

Reply:
done